app/javascript/moderationArticles/singleArticle/index.jsx

```diff
@@ -17,6 +17,42 @@ const SingleArticle = ({
     );
   });
 
+  const months = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ];
+
+  const get12HourTime = (date) => {
+    const minutes = date.getMinutes();
+    let hours = date.getHours();
+    const AmOrPm = hours >= 12 ? 'PM' : 'AM';
+    hours = hours % 12 || 12;
+
+    return `${hours}:${minutes} ${AmOrPm}`;
+  };
+
+  const formatDate = (timestamp) => {
+    const dateToday = new Date();
+    const origDatePublished = new Date(timestamp);
+
+    if (dateToday.toDateString === origDatePublished.toDateString) {
```

Comment:
This should be `dateToday.toDateString()` :)

Reply:
Lawd 🤦‍♀️